addon/utils/fetchData.js

```diff
@@ -67,7 +67,11 @@ WHERE {
     ?signUri org:classification ?signClassification.
   }
 }
-${count ? '' : `GROUP BY ?uri ?label ?template ?zonality\n ORDER BY ?label`}
+${
+  count
+    ? ''
+    : `GROUP BY ?uri ?label ?template ?zonality\n ORDER BY strlen(str(?label)) ?label`
```

Comment:
I think but the sign uri is included here? Couldn't you just have included a 
`ORDER BY ASC(COUNT(?signUri)) ?label` would have worked as well :)

Reply:
also: did you verify the default sort is ASCENDING ? probably best to make it explicit

Reply:
This gives me a 500 on the backend not sure why, and yes by default is ascendant but I made it explicit better